starlight_help/src/content/docs/import-from-microsoft-teams.mdx

```diff
@@ -22,6 +22,7 @@ The import will include your organization's:
 
 * **Name**
 * **Users**, including names, emails, roles
+* **Teams**, including all user subscriptions
```

Comment:
This probably should specify clearly that the Teams are converted into Zulip channels, right?